Update filesystem docs for GCS Fuse migration (Generations, rollout, path limits) (#10180)
pantheon-systems/documentation: src/source/content/guides/filesystem/01-introduction.md

- [Drupal 7](https://github.com/pantheon-systems/drops-7/blob/master/.gitignore)
- [WordPress](https://github.com/pantheon-systems/WordPress/blob/default/.gitignore)

<Alert title="Filesystem Migration in Progress" type="info">

> Pantheon is migrating sites from Valhalla (Generation 1) to Google Cloud Storage Fuse, or GCS Fuse (Generation 2). New sites created on or after August 3, 2026 use GCS Fuse by default. Existing sites are migrated in batches starting September 1, 2026, with completion targeted by the end of 2026. Migrations are managed by Pantheon and require no action for most sites. See [known limitations](/guides/filesystem/large-files#file-path-length) for details on the small number of sites that may be affected.

Reviewer: ```suggestion
Pantheon is migrating sites from Valhalla (Generation 1) to Google Cloud Storage FUSE, or gcsfuse, (Generation 2). New sites created on or after August 3, 2026 use gcsfuse by default. Existing sites are migrated in batches starting September 1, 2026, with completion targeted by the end of 2026. Migrations are managed by Pantheon and require no action for most sites. See [known limitations](/guides/filesystem/large-files#file-path-length) for details on the small number of sites that may be affected.
```

Google documentation styles this as "Cloud Storage FUSE" or "gcsfuse" (lowercase, software/repo).

It is never correct to partially capitalize the acronym as "Fuse".

References:
- https://docs.cloud.google.com/storage/docs/cloud-storage-fuse/overview
- https://github.com/googlecloudplatform/gcsfuse

</Alert>

## Files and Application Containers

The Pantheon architecture relies on highly available [application containers](/application-containers) that are seamlessly integrated with a cloud-based filesystem. This means that your files are not local to the application containers running your site's codebase.

> Pantheon refers to filesystem versions as **Generations**. Valhalla, Pantheon's original Cassandra-backed filesystem, is Generation 1. Google Cloud Storage Fuse, Pantheon's current filesystem, is Generation 2. Both may be in use across the platform during the migration period described above; the customer-facing experience is the same on either generation.

Reviewer: ```suggestion
Pantheon refers to filesystem versions as **Generations**. Valhalla, Pantheon's original filesystem, is Generation 1. Google Cloud Storage FUSE, Pantheon's current filesystem, is Generation 2. Both may be in use across the platform during the migration period described above; the customer-facing experience is the same on either generation.
```

nit: Introducing the word "Cassandra" here when we're getting rid of it may contribute to increased confusion.

Technically / pedantically, I object to the phrase "Cassandra-backed", as all Valhalla file content is stored in Cloud Storage buckets (just like gcsfuse!).

Also, applying the word "current" to the future filesystem when the original filesystem still exists may cause confusion.

> Your Generation 1 (Valhalla) or Generation 2 (GCS Fuse) filesystem creates a symbolic link (**symlink**) to the `files` directory in the appropriate location of your docroot:

Reviewer: ```suggestion
Your Generation 1 (Valhalla) or Generation 2 (gcsfuse) filesystem creates a symbolic link (**symlink**) to the `files` directory in the appropriate location of your docroot:
```
Suggestion, for capitalization consistency.

## Pantheon-Related Files

On Generation 1 (Valhalla) sites, Pantheon places the files below in your application container because they contain important information:

> **fusedav_version**: shows the version of fusedav being used.

Reviewer: Are you sure about this?  I don't recall seeing such a file, anywhere, ever.

Author: Reverted
